ui/src/adapters/parsers/forms.ts

```diff
@@ -126,6 +129,18 @@ const issueCredentialFormDataParser = getStrictParser<IssueCredentialFormData>()
     proofTypes: z
       .array(z.union([z.literal("MTP"), z.literal("SIG")]))
       .min(1, "At least one proof type is required"),
+    refreshService: z.union([
+      z.object({
+        enabled: z.literal(false),
+        url: z.string(),
```

Comment:
Why do we need `url` as string when `enabled` is false?

Reply:
Because it needs to adhere to the IssueCredentialFormData type that is how the form is defined.

Reply:
Good suggestion.  Changed